test/tracing/frcnn/test_frcnn_tracing.cpp

```diff
@@ -1,13 +1,13 @@
 #include <ATen/ATen.h>
 #include <torch/script.h>
 #include <torch/torch.h>
-#include <torchvision/roi_align.h>
 #include <torchvision/nms.h>
+#include <torchvision/roi_align.h>
```

Comment:
Are those individual imports needed or can we just include `torchvision/vision.h`?

Reply:
Including the nms is necessary because vision.h no longer includes all individual header files. I'm not sure that roi_align is necessary though. I left it because we had it before. Do you want me to try and remove it?

Reply:
Oh yeah. We should fix this linker issue soon so that we can get rid of those hacks.